test/e2e/testkit/mocks/logbackend_config_map.go

```diff
@@ -0,0 +1,102 @@
+//go:build e2e
+
+package mocks
+
+import (
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type LogBackendConfigMap struct {
```

Comment:
Let's split this to two types. It is used to create two different ConfigMaps that are only partially related.